package.json

```
@@ -35,7 +35,7 @@
     "prop-types": "^15.7.2"
   },
   "peerDependencies": {
-    "react-native": "^0.59.5",
+    "react-native": ">=0.59.5 <0.64.0",
     "react-native-windows": "^0.57.2"
   }
```

```suggestion
  },
  "peerDependenciesMeta": {
    "react-native-windows": {
      "optional": true
    }
  }
```

This will make sure that npm 7 does not automatically install `react-native-windows` when installing this package. Very important